esmvaltool_sample_data/__init__.py

```diff
@@ -87,6 +90,22 @@ def load_timeseries_cubes(mip_table: str = 'Amon') -> list:
     return list(cubes)
 
 
+def get_rootpaths() -> dict:
+    """Return a dict with rootpaths to update the user config in ESMValTool."""
+    rootpath = {
+        'rootpath': {
+            'CMIP6': [
+                str(base_dir / 'data' / 'timeseries' / 'CMIP6'),
+            ]
+        },
+        'drs': {
+            'CMIP6': 'default',
```

Comment:
`default` is a directory structure without any subdirectories, so this will probably not work

Reply:
It actually does work. I tested it and as long as the `.nc` file is in one of the subdirectories defined by the rootpath, it will find it.

Reply:
Probably not, there have been complaints about this: https://github.com/ESMValGroup/ESMValCore/issues/281

Reply:
Alright, I will change it to DKRZ then and make a note in the readme 👍